Makefile

```diff
@@ -498,6 +498,13 @@ ggml-mpi.o: ggml-mpi.c ggml-mpi.h
 	$(CC) $(CFLAGS) -c $< -o $@
 endif # LLAMA_MPI
 
+ifdef LLAMA_ARM_MMLA
+        MK_CPPFLAGS += -D__ARM_FEATURE_MMLA
+        MK_CFLAGS   += -D__ARM_FEATURE_MMLA
+        MK_CXXFLAGS += -D__ARM_FEATURE_MMLA
```

Comment:
The Makefile should use tabs for indentation, not spaces.

Reply:
thanks for the review. As @AGSaidi  mentioned I don't need this change, so, cleaned it up.